cli/native_deployment/native_deployment.go

```diff
@@ -311,6 +312,30 @@ func (p *NativeDeploymentConfigPlugin) StartTest(homeDirectory string, userInput
 		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("'%v %v' unable to get service dependencies, %v", dev.SERVICE_COMMAND, dev.SERVICE_START_COMMAND, derr))
 	}
 
+
+	// Add binds for the provided service secret files if the secret name is specified in a dependent service
```

Comment:
Move all this logic into a function, probably a stand-alone function in the dev package.

Reply:
It's in dev/secrets.go now